tutorials/blockchains/system-chains/.pages

```diff
@@ -0,0 +1,4 @@
+title: System Chain Tutorials
```

Comment:
```suggestion
title: System Chain Tutorials
```

I don't think tutorials is needed for the left nav as it already shows up under Blockchain Tutorials